cmd/waku/server/rest/health_api.yaml

```diff
@@ -0,0 +1,41 @@
+openapi: 3.0.3
```

Comment:
We should maintain these yml files for REST API's also in a common repo like `proto` files rather than each impl having a copy of them.
cc @NagyZoltanPeter  @jm-clius

Reply:
Tracking work for this in here: https://github.com/waku-org/go-waku/issues/739